src/components/Template.tsx

```diff
@@ -17,6 +17,9 @@ const useStyles = makeStyles({
 });
 
 const Template: React.FC = () => {
+  React.useEffect(() => {
```

Comment:
again, would be nice if we had a cleaner solution.

wait can we not just put this on the App.tsx

Reply:
Im thinking no because we need the code to run on component mount and the App component is always mounted while the router components appear/disappear